ledger-core/virtual/callregistry/objects_request_table.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package callregistry
+
+import (
+	"github.com/insolar/assured-ledger/ledger-core/reference"
+)
+
+// store request per object by object reference.
+type ObjectsRequestsTable struct {
+	knownRequests map[reference.Global]*CallResults
```

Comment:
If you will add method like 
```
func (ort *ObjectsRequestsTable) PutResults(ref, results)
```

then you don't need *CallResults over here. I think it would be better

Reply:
* and also allocates less

Reply:
CallResult need for wrap map in knownRequests.
```
knownRequests map[reference.Global]*CallResults
```
vs
```
knownRequests map[reference.Global]map[reference.Global]CallSummary
```

Reply:
@sexyhamster Please check new version